change/@microsoft-fast-components-c15c784b-9a60-441a-88dc-f032d990e5bb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
I don't think this is a major change; that would result in a breaking version change on publish. Can you downgrade this to minor?